UI/MapStatisticsView.cs

```diff
@@ -69,7 +74,66 @@ public string MapStatisticsString()
+
+      if (MapStatistics.NumberOfRegions > 0)
+      {
+        stats += $"{Environment.NewLine}";
+        foreach (var region in Settings.Regions.OrderBy(p=>p.RegionName).Where(p=>p.RegionName != Trizbort.Region.DefaultRegion))
+        {
+          if (MapStatistics.NumberOfRoomsInRegion(region.RegionName) == 0)
+            stats += $"{region.RegionName} has no rooms.";
+          else if (MapStatistics.NumberOfRoomsInRegion(region.RegionName) == 1)
+            stats += $"Only room in {region.RegionName}: ";
+          else
+            stats += $"List of rooms in {region.RegionName}: ";
+
+          var roomsInRegion = Project.Current.Elements.OfType<Room>().ToArray().OrderBy(p=>p.Name).Where(p=>p.Region == region.RegionName);
+          stats += string.Join(", ", roomsInRegion.Select(x => x.Name).ToArray()) + Environment.NewLine;
+        }
+        stats += $"{Environment.NewLine}";
+      }
+
+      if (MapStatistics.NumberOfRegions > 1)
+      {
+        stats += $"{Environment.NewLine}Region Links:{Environment.NewLine}";
+        var allRegions = Settings.Regions.OrderBy(p => p.RegionName).Where(p => p.RegionName != Trizbort.Region.DefaultRegion);
+        foreach (var region1 in allRegions)
+        {
+          var linkedRegions = string.Join(", ", Settings.Regions.OrderBy(p => p.RegionName).ToArray().Where(p => MapStatistics.RegionsLinked(region1, p)).Select(x => x.RegionName));
+
+          stats += $"{Environment.NewLine}";
+
+          if (linkedRegions == $"")
+            stats += $"{region1.RegionName} is not linked to any other regions.";
+          else
+            stats += $"{region1.RegionName} is linked to {linkedRegions}.";
+
+        }
+        stats += $"{Environment.NewLine}";
+      }
+
+      stats += string.Format("{0} up-down connection{1}{2}", MapStatistics.UpDown, plur(MapStatistics.UpDown), Environment.NewLine);
+      stats += string.Format("{0} in-out connection{1}{2}", MapStatistics.InOut, plur(MapStatistics.InOut), Environment.NewLine);
+      stats += string.Format("{0} miscellaneous {1}connection{2}", MapStatistics.MiscConnect, plur(MapStatistics.MiscConnect), Environment.NewLine);
+      stats += string.Format("{0} diagonal connection{1} (2-way){2}", MapStatistics.DiagonalConnects(2), plur(MapStatistics.DiagonalConnects(2)), Environment.NewLine);
+      stats += string.Format("{0} diagonal connection{1} (1-way){2}", MapStatistics.DiagonalConnects(1), plur(MapStatistics.DiagonalConnects(1)), Environment.NewLine);
+      stats += string.Format("{0} connections with middle text (1-way){2}", MapStatistics.HasMiddleText, plur(MapStatistics.HasMiddleText), Environment.NewLine);
+      stats += string.Format("{0} bent connection{1}, {2} with no text{3}", MapStatistics.BentConnects(true), plur(MapStatistics.BentConnects(true)),
+          MapStatistics.BentConnects(false), Environment.NewLine);
+      stats += Environment.NewLine;
```

Comment:
We have all these stats for connections labelled in different ways, but none for connections not labeled at all (unless I missed it somewhere)